spread/project.go

```diff
@@ -644,6 +654,12 @@ func Load(path string) (*Project, error) {
 			return nil, fmt.Errorf("%s is missing a summary", suite)
 		}
 
+		for _, fname := range suite.Artifacts {
```

Comment:
could we have a function to check the artifact paths? then we can reuse for tests, suites and projects